README.md

```diff
@@ -0,0 +1,22 @@
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/vimeo/go-clocks)](https://pkg.go.dev/github.com/vimeo/go-clocks)
```

Comment:
nit: Could we add the tests passing badge here as well?

Reply:
Done